samcli/commands/build/command.py

```diff
@@ -136,6 +137,16 @@
     help="Enabled parallel builds. Use this flag to build your AWS SAM template's functions and layers in parallel. "
     "By default the functions and layers are built in sequence",
 )
+@click.option(
+    "--mount-with",
+    "-mw",
+    type=click.Choice(MountMode.values(), case_sensitive=False),
+    default=MountMode.READ.value,
+    help="Optional. Specify mount mode for building functions/layers inside container. "
+    "If mount with write permissions, some files in source code directory may be changed/added by the build process. "
```

Comment:
Might be a better help text? 🤔 
```suggestion
    "If it is mounted with write permissions, some files in source code directory may be changed/added by the build process. "
```